docs/AboutUs.adoc

```diff
@@ -45,11 +45,11 @@ Responsibilities: Dev Ops + Threading
 
 '''
 
-=== Benson Meier
-image::yl_coder.jpg[width="150", align="left"]
-{empty}[http://github.com/yl-coder[github]] [<<johndoe#, portfolio>>]
+=== Kion Shi Rong
+image::cambioforma.png[width="150", align="left"]
```

Comment:
Can you add your photo to `docs/images/cambioforma.png`? I think we should just not merge #14 and update the About Us page individually.